Formula/garble.rb

```diff
@@ -17,6 +17,8 @@ class Garble < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "deb8dd3296bbb92ac22ae64c205a846cfa97631661dbc705be56d6d055355a1d"
   end
 
+  disable! date: "2023-07-15", because: :does_not_build
+
```

Comment:
Only temporary, to be removed
```suggestion
```

Reply:
re-enabled in 7bfd23840c445575b4bcf2779bc8d3b18eec535e after cherry picking 6d140c09b77dd33e8cfd2ddfe5354ea20f7582b1 from #137087